code/ui/InfoFeed.cs

```diff
@@ -8,20 +13,30 @@ public InfoFeed()
         {
             Current = this;
 
-            StyleSheet.Load( "/ui/InfoFeed.scss" );
+            StyleSheet.Load("/ui/InfoFeed.scss");
         }
 
-        public virtual Panel AddEntry(ulong leftSteamId, string leftName, ulong rightSteamId, string rightName, string method)
+        public virtual Panel AddEntry(Client leftClient, Client rightClient, string method, string append = null)
```

Comment:
Can we just make this an empty string by default?
`append = ""`

Also, I think we can find a better name for `append`, maybe `postfix`? or `following`

Reply:
If this is better now (ty for the hint, you are right), feel free to resolve and merge :)